### DISCUSSION TITLE
advertise this integration under https://www.home-assistant.io/integrations/#energy ?

### DISCUSSION THREAD

**Discussion by Hammi71** [TypQxQ/Sigenergy-Home-Assistant-Integration#3] [Ideas]
Hi there,
today famous german heise - magazine published an HowTo for integrating a Huawei-inverter into HomeAssistant under https://www.heise.de/ratgeber/Smart-Home-Daten-von-Huawei-Solaranlage-in-Home-Assistant-einbinden-9858374.html?giftToken=741f6803-ed13-41ad-91e6-ec80088c31c2.
This btw led me, after a short research, to this project.
Wouldn't it be good, to advertise this project under https://www.home-assistant.io/integrations/#energy ?
The currently rather small community here could maybe grow much faster that way.
best regards,
Marcus

**Comment by TypQxQ** [comment]
Hi,
That would be great. I unfortunately don't speak german so maybe you could?
I advertised this in the Sigenergy group on Facebook last week when it was taking shape.
Regards,
Andrei

**Reply by Hammi71** [reply]
Hi Andrei,

I assume a little missunderstanding. I only mentioned the heise-article, because it drove me to do an independent web-research, which led me to this project here. But I left a link to this project in the comment-section of the article : https://www.heise.de/forum/heise-online/Kommentare/Smart-Home-Daten-von-Huawei-Solaranlage-in-Home-Assistant-einbinden/Re-Welche-Wechselrichter-lassen-sich-direkt-einbinden/posting-44406384/show/

Could you provide the link to the Sigenergy group on Facebook, where you advertised this project ? I hope I can read it without a Facebook-account. It's btw a difficult relation between me and social networks, controlled by large profit-oriented companies.
That's why I appreciate this project so much, because I don't like my data lying in chinese clouds.

Best regards,
Marcus

**Comment by TypQxQ** [comment]
Also, to have it listed under https://www.home-assistant.io/integrations/#energy I believe it must be a real integration baked into Home Assistant, no yaml configuration, only frontend.
I could port this to Python and HACS installation, but I just wanted to do something quick that worked.

**Reply by Hammi71** [reply]
O.K. Maybe this project needs still some improvements, before porting it to the requested format for full integration.
